bw2io/extractors/ecospold1.py

```diff
@@ -21,6 +21,23 @@ def getattr2(obj, attr):
 class Ecospold1DataExtractor(object):
     @classmethod
     def extract(cls, path, db_name, use_mp=True):
+         """
+        Extracts data from ecospold1 files.
```

Comment:
Please use the imperative "Extract data from ..." for all functions. Please also check below.

Reply:
As advised, I've changed the phrasing in the docstrings for all functions to the imperative.